pkg/postage/stampissuer.go

```diff
@@ -110,3 +110,16 @@ func toString(buf []byte) string {
 	}
 	return string(buf[i:])
 }
+
+func (st *StampIssuer) Utilization() uint32 {
+	top := 0
+
+	//TODO return top bucket value for now
```

Comment:
fullness percentage can then be calculated as:

     max_bucket_value / 2 ^ (batch_depth - bucket_depth)